### PR TITLE
(195) Project tabs

### PR DESCRIPTION
**Pull request by jkempster34**
1. Add project subnavigation tabs
We want to use the GOV.UK tab component to navigate between the project task list page and the project information page. Move the "key information" into a partial, add partials to build the tab component, and add a basic controller and view for the new project information page.

2. Include authentication on the TasksController

[screen-capture (1).webm](https://user-images.githubusercontent.com/47089130/180465758-43689642-a63b-4637-83fc-f90b56b8ff94.webm)

### CODE REVIEW

**Comment by jkempster34** [comment]
[Add Project Information to Project view](https://trello.com/c/VS0BOMTz/195-add-project-information-to-project-view)

File: config/routes.rb [hunk 1]
```diff
@@ -17,4 +17,6 @@
   resources :projects do
     resources :tasks
   end
+
+  resources :project_information
```
Comment (jacksonj04) [inline config/routes.rb:21]: I think this would be better as `/project/information` rather than `/project_information`.

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
A little routing tweak I think to make the paths more intuitive, but otherwise looking good!

File: config/routes.rb [hunk 2]
```diff
@@ -17,4 +17,6 @@
   resources :projects do
     resources :tasks
   end
+
+  resources :project_information, path: "/projects/information"
```
Comment (jacksonj04) [inline config/routes.rb:21]: Should this be a child of the projects resource, so we get `projects/{id}/information"`?

**Review by jacksonj04 — APPROVED** [review]

**Pull request merged**